### PR TITLE
Add PHPStan checks (level 1)

### PR DESCRIPTION
**Pull request by cedric-anne**
Finalization of #37 based on shared workflow (https://github.com/glpi-project/plugin-ci-workflows/blob/v1/.github/workflows/continuous-integration.yml)

### CODE REVIEW

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -22,50 +21,9 @@ jobs:
...
-          key: "${{ github.job }}-${{ matrix.php-version }}-dependencies-${{ hashFiles('**/composer.lock') }}"
-      - name: "Install Composer dependencies"
-        run: |
-          composer install --ansi --no-interaction --no-progress --prefer-dist
-      - name: "PHP Parallel Lint"
-        run: |
-          vendor/bin/parallel-lint --colors --checkstyle --exclude ./vendor/ . | vendor/bin/cs2pr
-      - name: "PHP_CodeSniffer"
-        run: |
-          vendor/bin/phpcs -q --report=checkstyle | vendor/bin/cs2pr
```
Comment (trasher) [inline .github/workflows/continuous-integration.yml:62]: This one seems not running

**Review by trasher — CHANGES_REQUESTED** [review]
phpstan is not run

File: .github/workflows/continuous-integration.yml (hunk 1 above)
Comment (cedric-anne) [reply .github/workflows/continuous-integration.yml:62]: They are, but as output is piped to `cs2pr`, it is actually hard to know it. I will fix this.

**Comment by cedric-anne** [comment]
I removed usage to `cs2pr` for now and added some messages (in yellow in following screenshots).
image

**Pull request merged**